orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AuthCodeResponseGenerationService.java

```diff
@@ -71,18 +71,11 @@ public Map<String, String> getDimensions(
     }
 
     public void processVectorOfTrust(ClientSession clientSession, Map<String, String> dimensions) {
-        var mfaNotRequired =
-                clientSession
-                        .getEffectiveVectorOfTrust()
-                        .getCredentialTrustLevel()
-                        .equals(CredentialTrustLevel.LOW_LEVEL);
-        var levelOfConfidence = LevelOfConfidence.NONE.getValue();
-        if (clientSession.getEffectiveVectorOfTrust().containsLevelOfConfidence()) {
-            levelOfConfidence =
-                    clientSession.getEffectiveVectorOfTrust().getLevelOfConfidence().getValue();
-        }
```

Comment:
I don't think what we have now is going to return the same thing in the case of no level of confidence - the helper returns empty string, and here we return `LevelOfConfidence.NONE.getValue()` which is `P0`